gateways/helpers/sendRequest/sendRequest.go

```diff
@@ -0,0 +1,58 @@
+package sendRequest
```

Comment:
* Don't use `under_scores` or `mixedCaps` for package names - look this link https://blog.golang.org/package-names